controller/controller_manager.go

```diff
@@ -40,6 +40,8 @@ func StartControllers(stopCh chan struct{}, controllerID, serviceAccount, manage
 	if err != nil {
 		return nil, nil, errors.Wrap(err, "unable to get client config")
 	}
+	config.QPS = 100
```

Comment:
Use constants.

And what's the default value? Unless we know the impact, I would be cautious about changing those.

Reply:
Default QPS is 5, Burst is 10. The impact is more traffic to the apiserver, which results in heavier etcd usage. 100 QPS will probably generate too much etcd traffic for comfort, especially on large node clusters since longhorn-manager runs as a DaemonSet. Setting QPS to 10 or 20 should be pretty safe, and will reduce bottlenecks in attach that create/update lots of replica objects

I set QPS=100 to illustrate a point: the attach speed of a 100-replica volume with QPS=5 would take 3+ minutes, whereas QPS=100 would take seconds. The long term goal is to reduce the frequency of reads/writes as much as possible.